validator/project_validator.go

```diff
@@ -135,6 +137,12 @@ var projectErrorValidators = []projectValidator{
 	validateAliases,
 }
 
+// Functions used to validate the syntax of project configs representing properties found on the project page.
+var projectConfigErrorValidators = []projectConfigValidator{
```

Comment:
Wanted to make sure you saw this comment https://github.com/evergreen-ci/evergreen/pull/5371#pullrequestreview-875226345

Reply:
My bad! Thanks for bubbling it back up